test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -58,13 +59,17 @@ Feature: Bind an application to a service using annotations
                     kind: Backend
                     name: backend-demo
                     id: SBR
+                application:
+                    name: rsa-2-service
+                    group: apps
+                    version: v1
+                    resource: deployments
             """
-        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "True"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "binding-request-backend-a" should be changed to "EmptyApplication"
+
+        Then Service Binding "binding-request-backend-a" is ready
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-backend-a" should be changed to "True"
-        And Secret contains "BACKEND_READY" key with value "true"
-        And Secret contains "BACKEND_HOST" key with value "example.common"
```

Comment:
we should check if the application sees the appropriate bindings. Also scenario should be renamed to reflect the change in the usecase.

Reply:
Done. 5848c6ad9efcae4fd19a89610c4b87091efb3376